docs/dev/settings.rst

```diff
@@ -167,6 +167,22 @@ providers using the following environment variables:
 .. envvar:: RTD_SOCIALACCOUNT_PROVIDERS_GOOGLE_CLIENT_ID
 .. envvar:: RTD_SOCIALACCOUNT_PROVIDERS_GOOGLE_SECRET
 
+AWS configuration
+~~~~~~~~~~~~~~~~~
+
+The following variables can be used to use AWS in your local environment.
+Useful for testing :doc:`temporary credentials </aws-temporary-credentials>`.
+
+.. envvar:: RTD_S3_PROVIDER
+.. envvar:: RTD_AWS_ACCESS_KEY_ID
+.. envvar:: RTD_AWS_SECRET_ACCESS_KEY
+.. envvar:: RTD_AWS_STS_ASSUME_ROLE_ARN
```

Comment:
This changes the current semantic of the `RTD_` prefix.

We have been using `RTD_` prefix for settings specific for Read the Docs. These settings are for third-party applications, so I'd use a different prefix here. I suggest `DJANGO_` for all these settings.

Reply:
All environment variables that are passed to our application are prefixed with RTD, settings related to the platform are prefixed with RTD (settings.py).